File: TAREA1/alturamedia.java
import java.util.Scanner;

public class alturamedia {
    public static void main(String[] args){
        try(Scanner r=new Scanner (System.in)){
            //Declaramos variables
            int N=0;
            int AlturaSuperior=0;
            int AlturaInferior=0;
            int AlturaMedia=0;
            int AlturaCm=0;
            int sa=0;
            int X=0;
            System.out.println("INTRODUZCA UN VALOR PARA DEFINIR N: ");
            //Leer variable desde teclado
            N=r.nextInt();
            int arreglo[]=new int[N];
            //Utilizamos el ciclo do while par calcular la media de la altura de las N personas
            do{
                System.out.println("Ingrese su altura en cm: ");
                //Leer vatiable desde teclado.
                AlturaCm=r.nextInt();
                arreglo[X]=AlturaCm;
                sa=sa+arreglo[X];
                X++;
            }while(X<N);
            AlturaMedia=sa/X;
            //usamos el ciclo for para leer cada altura del arreglo
            for(int i=0; i<arreglo.length; i++){
                if(arreglo[i]>=AlturaMedia){
                    AlturaSuperior++;
                }else{
                    AlturaInferior++;
                }
            }
            //Mostramos por pantalla las alturas superiores a la media y las inferiores
            System.out.println("Las personas que tienen una altura superior a la media son: "+AlturaSuperior);
            System.out.println("Las personas que tienen una altura inferior a la media son: "+AlturaInferior);
        }
    }
}

File: EJERCICIOS_ARRAY/EJERCICIO 1/alturamedia.py
sumadealturas = 0
superiores = 0
inferiores = 0

n = int(input("Ingrese la cantidad de personas a las que va a pedir su altura: "))
alturas = []
for _ in range(n):
    altura = int(input("Ingrese la altura de la persona en cm: "))
    alturas.append(altura)
    sumadealturas += altura

alturamedia = sumadealturas / n

for altura in alturas:
    if altura <= alturamedia:
        superiores += 1
    else:
        inferiores += 1

print(superiores, "personas tienen una altura superior a la media")
print(inferiores, "personas tienen una altura inferior a la media")